frontstage/templates/account/account.html

```diff
@@ -13,17 +13,8 @@
 ] %}
 
 {% block main %}
-{% with messages = get_flashed_messages() %}
-    {% if messages %}
-          <div class="panel panel--simple panel--error u-mb-m">
-              <div class="panel__body" data-qa="error-body">
-                {% for message in messages %}
-                <p class="u-fs-r" id="flashed-message-{{ loop.index }}">{{ message }}</p>
-                {% endfor %}
-              </div>
-          </div>
-    {% endif %}
-{% endwith %}
+{% set ns = namespace () %}
```

Comment:
What is this?  Also, if it's a function then there shouldn't be a space between the word and the brackets

Reply:
Yes its a function. I am declaring a global variable using namespace. You are right there should not be spaces. Making the change.